pyprep/ransac.py

```diff
@@ -127,7 +131,7 @@ def find_bad_by_ransac(
         random_ch_picks.append(picks)
 
     # Correlation windows setup
-    correlation_frames = corr_window_secs * sample_rate
+    correlation_frames = int(corr_window_secs * sample_rate)
```

Comment:
Why is this change necessary? If it *is* necessary, please also change the other parts in the code where `correlation_frames` is calculated - e.g., line 294 in find_noisy_channels.py

Reply:
That was just so I could pass `correlation_frames` as an argument to `_ransac_correlations_alt` instead of `n`, since I thought it was more readable (it's used for array slicing, so it needs to be an int). I can change it elsewhere as well.

Reply:
We just need to be sure about the rounding logic here. So that with the new change we don't accidentally change a behavior that was working before. See below:

```
In [8]: np.arange(5.5)
Out[8]: array([0., 1., 2., 3., 4., 5.])

In [9]: np.arange(int(5.5))
Out[9]: array([0, 1, 2, 3, 4])
```